templates/profiles.html

```diff
@@ -311,6 +337,18 @@ <h4 id="name" style="margin:5px; margin-top:5px;"></h4>
         data['tactic'] = $('#ability-tactic-name').val();
         data['technique'] = {'attack_id': $('#ability-tech-id').val(), 'name': $('#ability-tech-name').val()};
         data['platforms'] = platforms;
+
+        $('#ability-additional-info').find('#additional-info-table tr').each(function() {
+            let key = $(this).find('.additional-key').val();
+            if (key !== ''){
+                if (key in data){
+                    alert('Cannot save additional field! \'' + key + '\' in additional info is a reserved keyword.');
```

Comment:
don't alert, use the alert modal that we use elsewhere

Reply:
Using the duk-modal instead of alert. I hope that's the one you meant...